### ISSUE TITLE
The best resolution for HE when using pre-trained model

### ISSUE THREAD

**Issue by ZengTaox** [gevaertlab/sequoia-pub#5]
I have participated in some pathology-related spatial metabolome and spatial transcriptome studies and found that the signal could be different under different resolution. Therefore, I want to know whether the accuracy of model prediction and the accuracy of gene spatial information could be different under different resolution and the "work resolution" of the pre-trained model.

**Comment by YuanningEric** [comment]
Hi! The ideal image resolution should be 0.5um/pixel.

**Comment by mpizurica** [comment]
Indeed, as Yuanning mentioned, the ideal resolution (which the model was trained on) should be 0.5um/pixel. 

Generally if resolution decreases, model performance is expected to be worse. Especially in currently available spatial transcriptomics studies the resolution of the WSIs seems to be quite bad (as we address in the paper and as we showed in Supplementary). Hence such spatial data is not really ideal to evaluate the model on, because the model cannot show its full potential on the low-resolution WSI.
